scss/_button-group.scss

```diff
@@ -20,6 +20,12 @@
   > .btn.active {
     z-index: 1;
   }
+
+  // Boosted mod
+  &:focus-within {
```

Comment:
Could you please explain to me this use case?

Reply:
This is for the click on a button `:active`, it overlays the siblings and it removes the border issue.